### DISCUSSION TITLE
Budibase user table not showing any users in app

### DISCUSSION THREAD

**Discussion by Bong1hong** [Budibase/budibase#8919] [Help] [answered]
Hi All,

Currently, I have encounter a issue where the Budibase user table does not load any user in the app.

First, it load for few minutes (< ~3 minutes).
![loading-user-table](https://user-images.githubusercontent.com/18653633/205428897-bb2192f9-de1c-4ffb-ae82-dbdaa0c0d3ae.PNG)

Then it return empty user table (even though there's users that can access the app).
![empty-user-table](https://user-images.githubusercontent.com/18653633/205428900-0ef60649-fc6e-4615-91f3-4feff5e870ce.png)

Additional Information:

- Hosting Method: Self-hosted
- Budibase version: 2.1.28

Thanks for any helps

**Comment by melohagan** [comment]
Hey @Bong1hong 

Can you show me (feel free to obfuscate) the users assigned to the app. Do you see any errors in the console/network tabs? (Fn+F12 on Chrome)

**Reply by Bong1hong** [reply]
Hi @melohagan,

There are the assigned users to the app (email hidden):
![access-list](https://user-images.githubusercontent.com/18653633/206329397-690bd9e6-8500-422c-9c43-92ec867a3371.PNG)

When I open the Users table, this is the console warning that appear:
![console-error](https://user-images.githubusercontent.com/18653633/206329143-36c2b563-b90e-41e7-8a31-d48e54d74904.PNG)

In the network tab, api/ta_users/search returns no rows as show below:
![network-return-nothing](https://user-images.githubusercontent.com/18653633/206329269-670d7441-a9c5-4abf-9484-0b8e84828e89.PNG)

Just to point out as well, below are the custom column added (unsure whether this has effect on the issue):
![custom-column](https://user-images.githubusercontent.com/18653633/206329827-40dfbb73-dca5-4227-a52c-c0604051f394.png)

Thanks for any helps

**Reply by melohagan** [reply]
There's no further detail in the `search` response tab about the error at all? 
It might help to have the app export if you want to email it to support@budibase.com

I'm also wondering when you added the custom columns. Did you do that before or after granting users access to your app?

Was this something that had previously been working but stopped working with an upgrade to **2.1.28**?

**Reply by Bong1hong** [reply]
Hi,

Thanks for your reply

1. There's no further detail in the search response tab.
2. I added custom columns **AFTER** granting some users access to the app.
3. Yes, previously it was in **2.1.24** and was working fine.

I will see if I have permission to export the App.

Thanks

**Comment by Bong1hong** [comment, marked as answer]
Hi, the issue is unexpectedly fixed **AFTER** I update to **v2.2.3** and revert back to **v2.1.28**

**Reply by melohagan** [reply]
@Bong1hong was it still broken in **v2.2.3** ? If you updated again would it now be resolved in the latest version?
